ui/core/src/model/dashboard.ts

```diff
@@ -38,3 +38,5 @@ export interface DashboardSelector {
   project: string;
   dashboard: string;
 }
+
+export type OnSaveDashboard = (dashboard: DashboardResource) => Promise<unknown>;
```

Comment:
Probably this should be define directly in the dashboard package. That doesn't sound it's part of the model for me.

Reply:
Yeahh went back and forth on that, moved to dashboards package now, [this file](https://github.com/perses/perses/pull/1210/commits/347f18154cdb8f5878578eb83d59bc0f26d5e235) maybe makes sense and has some utils that don't hurt to export out